libraries/plugins/delayed_node/delayed_node_plugin.cpp

```diff
@@ -58,7 +58,7 @@ delayed_node_plugin::~delayed_node_plugin()
 void delayed_node_plugin::plugin_set_program_options(bpo::options_description& cli, bpo::options_description& cfg)
 {
    cli.add_options()
-         ("trusted-node", boost::program_options::value<std::string>(), "RPC endpoint of a trusted validating node (required for delayed_node)")
+         ("trusted-node", boost::program_options::value<std::string>()->default_value("127.0.0.1:8090"), "RPC endpoint of a trusted validating node (required for delayed_node)")
```

Comment:
This is probably a bad default because that's usually the port where the node itself will be listening. IMO it's better to leave this option without a default. Users should have to actively choose which node they trust.

Reply:
I changed it back at https://github.com/bitshares/bitshares-core/pull/1437/commits/e198552d0dd1a36dd68067b16195dc4706bc740c

I added it because the binary seg faults when starting with no option:

```
alfredo@alfredo-System-Product-Name:~/CLionProjects/pull1427_2$ ./programs/delayed_node/delayed_node 
Segmentation fault (core dumped)
alfredo@alfredo-System-Product-Name:~/CLionProjects/pull1427_2$ 
```

It need to be started with trusted node option:

```
alfredo@alfredo-System-Product-Name:~/CLionProjects/pull1427_2$ ./programs/delayed_node/delayed_node --trusted-node "127.0.0.1:8090"
2348175ms th_a       main.cpp:111                  main                 ] Writing new config file at /home/alfredo/CLionProjects/pull1427_2/delayed_node_data_dir/config.ini
2348180ms th_a       db_management.cpp:174         open                 ] Wiping object_database due to missing or wrong version
2348180ms th_a       object_database.cpp:93        wipe                 ] Wiping object database...
2348180ms th_a       object_database.cpp:95        wipe                 ] Done wiping object databse.
```

This problem is independent from the changes in this pull request. Another example of segfault in the `delayed_node` binary can be observed at the end of the output from `./delayed_node --help`

As a workaround i tried: `app::application* node = new app::application();` before but was later discarded. 

I think we can address this issue in a separate pull request in order to dont complicate things too much further. 

Let me know what do you think, thanks again.

Reply:
The segfault will be fixed with #1529. It happens because the delayed_node plugin complains about the missing option and then tries to close a database that has never been opened.

Reply:
great, thanks.